cosmos/aggevm.json

```diff
@@ -0,0 +1,55 @@
+{
+  "rpc": "https://aggevm-rpc.web3-idea.xyz",
+  "rest": "https://aggevm-api.web3-idea.xyz",
+  "nodeProvider": {
+    "name": "Web3 Idea",
+    "email": "zhangzhishun1024@gmail.com",
+    "website":"https://web3-idea.xyz"
+  },
+  "chainId": "aggevm",
+  "chainName": "Agg",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "AGG",
+    "coinMinimalDenom": "uagg",
+    "coinDecimals": 6,
+    "coinGeckoId": "agg",
```

Comment:
@zhangzhishun Could you remove `coinGeckoId` field? because It is invalid

Reply:
Thanks.

But I received the message: "stakeCurrency.coinDenom" is required and "stakeCurrency.coinDenom" is not allowed to be empty after removing it.

Reply:
@zhangzhishun It is ok that you remove `coinGeckoId`field!